pallets/phala/src/compute/basepool.rs

```diff
@@ -144,51 +147,71 @@ pub mod pallet {
 		InvalidWithdrawalAmount,
 		/// RMRK errors
 		RmrkError,
-
+		/// The Specific pid does not match to any pools
 		PoolDoesNotExist,
-
+		/// Occurs when tring to get a stakepool from a PoolProxy contains a vault or vault from a stakepool instead.
 		PoolTypeNotMatch,
-
+		/// NftId does not match any nft
 		NoAvailableNftId,
-
+		/// Occurs when pool's shares is zero
 		InvalidSharePrice,
-
+		/// Try to get a NftGuard when the nft's spinlock is active
 		AttrLocked,
 	}
 
 	#[derive(Encode, Decode, TypeInfo, Clone, PartialEq, Eq, RuntimeDebug)]
 	pub struct BasePool<AccountId, Balance> {
+		/// Pool ID
 		pub pid: u64,
-
+		/// The owner of the pool
 		pub owner: AccountId,
-
+		/// Total shares
+		///
+		/// It tracks the total number of shared of all the contributors. Guaranteed to be
+		/// non-dust.
 		pub total_shares: Balance,
-
+		/// It tracks the current asset value owned by the pool
+		///
+		/// Including:
+		/// 1.Freestakes of the pool (both case)
```

Comment:
```suggestion
		/// 1. Freestakes of the pool (both case)
```

Careful about spacing as mentioned above.

Reply:
done